src/sage/misc/latex.py

```diff
@@ -583,6 +583,10 @@ def latex_extra_preamble():
         \newcommand{\PSL}{\mathrm{PSL}}
         \newcommand{\lcm}{\mathop{\operatorname{lcm}}}
         \newcommand{\dist}{\mathrm{dist}}
+        \newcommand{\im}{\mathop{\operatorname{im}}}
+        \newcommand{\rank}{\mathop{\operatorname{rank}}}
+        \newcommand{\PP}{\mathbf{P}}
```

Comment:
Should this be using the `\Bold` macro, like `\ZZ` does?

Reply:
Yes. Thanks.